<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../content.qrc">
    <normaloff>:/login/images/icons8_login.ico</normaloff>:/login/images/icons8_login.ico</iconset>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="driverListLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>125</height>
        </size>
       </property>
       <property name="text">
        <string>Database driver:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DriverList">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>2</number>
       </property>
       <property name="maxCount">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string notr="true" extracomment="QODBC">SQL SERVER</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="serverLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Server name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ServerNameLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ServerConnectionayout">
     <item>
      <widget class="QPushButton" name="ConnectServerButton">
       <property name="text">
        <string>Connect to server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DisconnectServerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="databaseListLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>125</height>
        </size>
       </property>
       <property name="text">
        <string>Select database:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DatabaseList">
       <property name="maxVisibleItems">
        <number>6</number>
       </property>
       <property name="maxCount">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="usernameLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>User name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="UsernameLine">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="passwordLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>User password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordLine">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ConnectDBButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Connect to DB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../content.qrc"/>
 </resources>
 <connections/>
</ui>
